docs/self-hosted/install/install-on-aws-script.md

```diff
@@ -21,6 +21,7 @@ export AWS_SECRET_ACCESS_KEY=...
 
 ## 2. Run the installer image
```

Comment:
Do we want to mention that the installer are basically broken right now?

Reply:
Awh, yes.